docs/drivers.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Driver Requirements
+---
+
+Connectors built with the Connector SDK use either an ODBC or a JDBC driver to communicate with the database.
+
+In general, we prefer JDBC connectors to ODBC connectors, as JDBC is cross-platform.
+
+
+# JDBC Drivers
+To learn more about JDBC drivers and Tableau, check *Specify the right JDBC driver* section in  the  [user documentation](https://help.tableau.com/current/pro/desktop/en-us/examples_otherdatabases_jdbc.htm).
```

Comment:
https://help.tableau.com/current/pro/desktop/en-us/examples_otherdatabases_jdbc.htm#specify-the-right-jdbc-driver
The anchor links do work, but maybe they lose too much context of the doc that way.